plugins/source-github/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/maximilien/kn-source-github
+
+go 1.13
+
+require (
+	github.com/maximilien/kn-source-pkg v0.4.0
+	github.com/spf13/cobra v1.0.0 // indirect
+	knative.dev/eventing-contrib v0.14.0
+	knative.dev/test-infra v0.0.0-20200413202711-9cf64fb1b912
```

Comment:
Why do you need test-infra ? We are including the (shared among all plugin) test-infra scripts in the top-level "test-infra" directory. Shared, because all plugins share the same test infra structure.

Plugins must not include test-infra anymore. See also the discussions at https://github.com/knative/test-infra/issues/2068

Reply:
Yeah, I can remove test-infa. I think I left it there as I copied the `./hack/build.sh` from client initially.